Errors when building godot with `disable_3d=yes`

### Tested versions

- Reproducible in current master https://github.com/godotengine/godot/commit/b377562b52fa8a68a99d5e818d8e08ac8d3fd7da
- Not reproducible in 4.4.stable

### System information

Windows 11

### Issue description

I’m getting some errors when building with the following command: scons platform=android target=template_debug arch=arm64 disable_3d=yes

```command
ERROR: scene\debugger\scene_debugger.cpp:1644:43: error: use of undeclared identifier 'selected_3d_nodes'; did you mean 'selected_ci_nodes'?
                                if (!selected_ci_nodes.is_empty() || !selected_3d_nodes.is_empty()) {
                                                                      ^~~~~~~~~~~~~~~~~
                                                                      selected_ci_nodes
scene\debugger/scene_debugger.h:242:24: note: 'selected_ci_nodes' declared here
        LocalVector<ObjectID> selected_ci_nodes;
                              ^
scene\debugger\scene_debugger.cpp:1713:64: error: use of undeclared identifier 'selected_3d_nodes'; did you mean 'selected_ci_nodes'?
        const int limit = max_selection - (selected_ci_nodes.size() + selected_3d_nodes.size());
                                                                      ^~~~~~~~~~~~~~~~~
                                                                      selected_ci_nodes
scene\debugger/scene_debugger.h:242:24: note: 'selected_ci_nodes' declared here
        LocalVector<ObjectID> selected_ci_nodes;
                              ^
scene\debugger\scene_debugger.cpp:1851:65: error: use of undeclared identifier 'nodes_3d'; did you mean 'nodes_ci'?
        if (!changed && nodes_ci.size() == selected_ci_nodes.size() && nodes_3d.size() == selected_3d_nodes.size()) {
                                                                       ^~~~~~~~
                                                                       nodes_ci
scene\debugger\scene_debugger.cpp:1790:24: note: 'nodes_ci' declared here
        LocalVector<ObjectID> nodes_ci;
                              ^
scene\debugger\scene_debugger.cpp:1851:84: error: use of undeclared identifier 'selected_3d_nodes'; did you mean 'selected_ci_nodes'?
        if (!changed && nodes_ci.size() == selected_ci_nodes.size() && nodes_3d.size() == selected_3d_nodes.size()) {
                                                                                          ^~~~~~~~~~~~~~~~~
                                                                                          selected_ci_nodes
scene\debugger/scene_debugger.h:242:24: note: 'selected_ci_nodes' declared here
        LocalVector<ObjectID> selected_ci_nodes;
                              ^
4 errors generated.
```

### Steps to reproduce

1. Clone master https://github.com/godotengine/godot/commit/b377562b52fa8a68a99d5e818d8e08ac8d3fd7da
2. Build with `scons platform=android target=template_debug arch=arm64 disable_3d=yes`
3. Wait for it.

### Minimal reproduction project (MRP)

N/A

--- a/scene/debugger/scene_debugger.cpp
+++ b/scene/debugger/scene_debugger.cpp
@@ -1641,7 +1641,11 @@ void RuntimeNodeSelect::_physics_frame() {
 				// Allow forcing box selection when an item was clicked.
 				selection_drag_state = SELECTION_DRAG_MOVE;
 			} else if (items.is_empty()) {
+#ifdef _3D_DISABLED
+				if (!selected_ci_nodes.is_empty()) {
+#else
 				if (!selected_ci_nodes.is_empty() || !selected_3d_nodes.is_empty()) {
+#endif // _3D_DISABLED
 					EngineDebugger::get_singleton()->send_message("remote_nothing_clicked", Array());
 					_clear_selection();
 				}
@@ -1710,7 +1714,10 @@ void RuntimeNodeSelect::_send_ids(const Vector<Node *> &p_picked_nodes, bool p_i
 		return;
 	}
 
-	const int limit = max_selection - (selected_ci_nodes.size() + selected_3d_nodes.size());
+	int limit = max_selection - selected_ci_nodes.size();
+#ifndef _3D_DISABLED
+	limit -= selected_3d_nodes.size();
+#endif // _3D_DISABLED
 	if (limit <= 0) {
 		return;
 	}
@@ -1848,9 +1855,15 @@ void RuntimeNodeSelect::_set_selected_nodes(const Vector<Node *> &p_nodes) {
 		}
 	}
 
+#ifdef _3D_DISABLED
+	if (!changed && nodes_ci.size() == selected_ci_nodes.size()) {
+		return;
+	}
+#else
 	if (!changed && nodes_ci.size() == selected_ci_nodes.size() && nodes_3d.size() == selected_3d_nodes.size()) {
 		return;
 	}
+#endif // _3D_DISABLED
 
 	_clear_selection();
 	selected_ci_nodes = nodes_ci;

